app/assets/javascripts/assignment_form.js

```diff
@@ -0,0 +1,7 @@
+function import_options(starter_code_field_value) {
+  if (starter_code_field_value.length > 0) {
+    document.getElementById('import-options').style.display ="";
+  } else {
+    document.getElementById('import-options').style.display ="none";
```

Comment:
A few style nitpicks: could use `'` or `"` consistently, and add a space after each `=`. What do you think about pulling out `const importOptions = document.getElementById('import-options');` to reduce the line length on lines 3 and 5?